WSL/toc.yml

```diff
@@ -4,18 +4,22 @@
   items:
     - name: What is WSL?
       href: about.md
+    - name: Best practices for set up
+      href: setup/environment.md
     - name: Comparing WSL 2 and WSL 1
       href: compare-versions.md
-- name: Quickstart
+- name: Install
   items:
-    - name: Install WSL & update to WSL 2
-      href: install-win10.md
-    - name: Best practices for setting up
-      href: setup/environment.md
+    - name: Install command
```

Comment:
I think having this many install pages is a bit overwhelming for users. I'd rather just have one really easy: "Getting Started" page that highlights --install, which will be the relevant page for 90% of users. Then we can have links on that page for users who are on older versions, etc.

Reply:
I think we should retitle this just to `Getting Started` or `Install WSL`.

Reply:
Messaged about this in Teams.. more users search for "install" than "get started" according to our search data. Made some updates and look forward to your feedback on them.